### PR TITLE
Aaron/translate landing page

### PR DESCRIPTION
**Pull request by aaronskiba**
This PR enables the landing page content to again render in French.
- fix issue #406 
- fix issue #408

### CODE REVIEW

**Comment by aaronskiba** [comment]
NOTE: Currently, the translation is only available in dmp-assistant-test (be sure to use the test key).

File: app/views/home/_welcome.html.erb [hunk 1]
```diff
@@ -62,13 +62,13 @@
...
       <p>
-        For more resources and training materials spanning the entire research data life cycle, see the <a href="https://alliancecan.ca/en/services/research-data-management/learning-and-training/training-resources/"> Portage Network Training Resources. </a>
+        For more resources and training materials spanning the entire research data life cycle, see the <a href="%{training_resources_path}"> Portage Network Training Resources. </a>
       </p>
 
       <p>
         The DMP Assistant was adapted from the <a href="https://dcc.ac.uk/"> Digital Curation Centre (DCC)</a>’s <a href="https://dmponline.dcc.ac.uk/">DMPonline</a> tool, and uses the DMP Roadmap codebase developed by DCC and the <a href="https://cdlib.org/services/uc3/">University of California Curation Center (UC3).</a>
       </p>
-    })
+    }),manage_data_path: how_to_manage_your_data_path, training_resources_path: training_resources_path)
```
Comment (lagoan) [inline app/views/home/_welcome.html.erb]: It is a very minor detail, we should keep consistency and change ```manage_data_path``` to ```how_to_manage_your_data_path``` on lines 71 and 35.

**Review by lagoan — CHANGES_REQUESTED** [review]
Work is looking good, its great this fixes the bug stopping the front page French translation.

I have a slight change suggestion for consistency.

Otherwise, it looks great!

File: app/views/home/_welcome.html.erb (hunk 1 above)
Comment (aaronskiba) [reply app/views/home/_welcome.html.erb]: Sounds good. Will fix that and then run the translation:sync command with prod key.

**Review by lagoan — APPROVED** [review]
LGTM

**Pull request merged**